libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -175,11 +201,25 @@ export function serialize(props: DynamicallyAddedParameterProps[]): ValueSegment
       return { ...resultPropertiesObj, [schemaKey]: propertyValue };
     }, {});
 
-  const rootObject = {
-    type: 'object',
-    properties,
-    required: requiredArray,
-  };
+  let rootObject: any;
```

Comment:
We should have a type for this or create one, and prefer no using `any` since this is new code when we should have the most context.
Maybe `SwaggerSchema`?

Reply:
Changed to type `Schema`